tests/unit.test.ts

```diff
@@ -1,4 +1,5 @@
 import {isNumber, formatTimeItem, validateTimeAndCursor} from '../src/index';
+import validate = WebAssembly.validate;
```

Comment:
What is this line for? It seems not to be used in the file.

Reply:
It's a useless line, i'll change that